packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -464,7 +464,13 @@ export function injectIntoGlobalHook(globalObject: any): void {
         onCommitFiberUnmount() {},
       };
     }
-
+    // Checks if DevTools hook is disabled
+    if (hook.isDisabled) {
+      console['warn'](
+        'Something has shimmed the React DevTools global hook (REACT_DEVTOOLS_GLOBAL_HOOK).' +
+          'Fast Refresh is not compatible with this shim and will be disabled.  ',
+      );
```

Comment:
Your spacing is a little off. This will print the string:
```
"Something has shimmed the React DevTools global hook (REACT_DEVTOOLS_GLOBAL_HOOK).Fast Refresh is not compatible with this shim and will be disabled.  "
```

```suggestion
      console['warn'](
        'Something has shimmed the React DevTools global hook (REACT_DEVTOOLS_GLOBAL_HOOK). ' +
          'Fast Refresh is not compatible with this shim and will be disabled.',
      );
```

Reply:
@bvaughn can you please review my pull request. I've implemented the suggested changes.

Reply:
GitHub notifies me when you push commits to a PR I'm reviewing, for what it's worth. You don't need to remind me. :)